frontend/packages/dev-console/src/components/health-checks/health-checks-types.ts

```diff
@@ -0,0 +1,39 @@
+import { NameValuePair } from '@console/shared';
+
+export enum HealthChecksProbeType {
+  ReadinessProbe = 'readinessProbe',
+  LivenessProbe = 'livenessProbe',
+  StartupProbe = 'startupProbe',
+}
+
+export enum ProbeType {
```

Comment:
What is the difference between `ProbeType` and `HealthChecksProbeType` both seems to be probe types.

Reply:
changed `ProbeType` to `RequestType`